WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -140,30 +140,27 @@ protected function setup_groups( $key ) {
 	 * @return void
 	 */
 	public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
-		$tokens = $phpcsFile->getTokens();
-		$token  = $tokens[ $stackPtr ];
+		$tokens        = $phpcsFile->getTokens();
+		$token         = $tokens[ $stackPtr ];
+		$token_content = strtolower( $token['content'] );
 
 		// Exclude function definitions, class methods, and namespaced calls.
-		if (
-			T_STRING === $token['code']
-			&&
-			( $prev = $phpcsFile->findPrevious( T_WHITESPACE, ( $stackPtr - 1 ), null, true ) )
-			&&
-			(
-				// Skip sniffing if calling a method, or on function definitions.
-				in_array( $tokens[ $prev ]['code'], array( T_FUNCTION, T_DOUBLE_COLON, T_OBJECT_OPERATOR ), true )
-				||
-				(
-					// Skip namespaced functions, ie: \foo\bar() not \bar().
-					T_NS_SEPARATOR === $tokens[ $prev ]['code']
-					&&
-					( $pprev = $phpcsFile->findPrevious( T_WHITESPACE, ( $prev - 1 ), null, true ) )
-					&&
-					T_STRING === $tokens[ $pprev ]['code']
-				)
-			)
-			) {
-			return;
+		if ( T_STRING === $token['code'] && isset( $tokens[ ( $stackPtr - 1 ) ] ) ) {
+			$prev = $phpcsFile->findPrevious( PHP_CodeSniffer_Tokens::$emptyTokens, ( $stackPtr - 1 ), null, true );
+
+			if ( false !== $prev ) {
+				// Skip sniffing if calling a same-named method, or on function definitions.
+				if ( in_array( $tokens[ $prev ]['code'], array( T_FUNCTION, T_DOUBLE_COLON, T_OBJECT_OPERATOR ), true ) ) {
+					return;
+				}
+
+				// Skip namespaced functions, ie: \foo\bar() not \bar().
+				$pprev = $phpcsFile->findPrevious( PHP_CodeSniffer_Tokens::$emptyTokens, ( $prev - 1 ), null, true );
+				if ( false !== $pprev && T_NS_SEPARATOR === $tokens[ $prev ]['code'] && T_STRING === $tokens[ $pprev ]['code'] ) {
+					return;
+				}
+			}
+			unset( $prev, $pprev );
```

Comment:
Correct, but that doesn't matter for an `unset()`. No notices will be thrown and if the variable *is* set, it will be correctly unset.